Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:15.5-alpine3.10 as build-deps
```

Comment:
Good stuff. Does the README have instruction about running this image (even if it is something as simple as `docker run <image name>`)?

Reply:
Add the docker instruction in README.